cves/CVE-2017-5664.yml

```diff
@@ -168,38 +173,41 @@ lessons:
     that one or two of them apply.
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
-  defense_in_depth:
-    applies:
+  defense_in_depth: 
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
+    applies: false
     note:
   security_by_obscurity:
-    applies:
+    applies: false
     note:
   serial_killer:
-    applies:
-    note:
+    applies: true
+    note: |
+      A hacker could use this to as a way to propose a denial of service,
+      incorrect error pages will result in hackers with their own pages that 
+      can cause the page to be stuck in a fixed position
```

Comment:
I'm having trouble understanding this sentence: "incorrect error pages will result in hackers with their own pages that can cause the page to be stuck in a fixed position". What do you mean "hackers with their own page"? Which page is "stuck" here? The sentence is also grammatically incorrect, which makes it difficult to understand as well.